frontends/election-manager/src/screens/print_test_deck_screen.test.tsx

```diff
@@ -47,15 +119,218 @@ test('Printing all L&A packages sorts precincts', async () => {
     'Southwest Ackerman',
     'West Weir',
   ];
+
   for (const [i, precinct] of precinctsInAlphabeticalOrder.entries()) {
-    const printText = `Printing L&A Package (${i + 1} of 13): ${precinct}.`;
-    await screen.findByLabelText(printText);
-    jest.advanceTimersByTime(5000);
-    await screen.findByLabelText(printText);
-    jest.advanceTimersByTime(30000);
-    await screen.findByLabelText(printText);
-    jest.advanceTimersByTime(30000);
+    const printText = `Printing L&A Package (${i + 1} of 13): ${precinct}`;
+    await screen.findByText(printText);
+    await waitFor(() =>
+      expect(mockPrinter.print).toHaveBeenCalledTimes(3 * i + 1)
+    );
+    expect(mockPrinter.print).toHaveBeenLastCalledWith({ sides: 'one-sided' });
+    await waitFor(() =>
+      expect(mockKiosk.log).toHaveBeenLastCalledWith(
+        expect.stringContaining(LogEventId.TestDeckTallyReportPrinted)
+      )
+    );
+    jest.advanceTimersByTime(ONE_SIDED_PAGE_PRINT_TIME_MS + 1);
+
+    await screen.findByText(printText);
+    await waitFor(() =>
+      expect(mockPrinter.print).toHaveBeenCalledTimes(3 * i + 2)
+    );
+    expect(mockPrinter.print).toHaveBeenLastCalledWith({ sides: 'one-sided' });
+    await waitFor(() =>
+      expect(mockKiosk.log).toHaveBeenLastCalledWith(
+        expect.stringContaining(LogEventId.TestDeckPrinted)
+      )
+    );
+    expect(mockKiosk.log).toHaveBeenLastCalledWith(
+      expect.stringContaining('BMD paper ballot test deck printed')
+    );
+    jest.advanceTimersByTime(3 * ONE_SIDED_PAGE_PRINT_TIME_MS + 1);
+
+    await screen.findByText(printText);
+    await waitFor(() =>
+      expect(mockPrinter.print).toHaveBeenCalledTimes(3 * i + 3)
+    );
+    expect(mockPrinter.print).toHaveBeenLastCalledWith({
+      sides: 'two-sided-long-edge',
+    });
+    await waitFor(() =>
+      expect(mockKiosk.log).toHaveBeenLastCalledWith(
+        expect.stringContaining(LogEventId.TestDeckPrinted)
+      )
+    );
+    expect(mockKiosk.log).toHaveBeenLastCalledWith(
+      expect.stringContaining('Hand-marked paper ballot test deck printed')
+    );
+    if (i < precinctsInAlphabeticalOrder.length - 1) {
+      jest.advanceTimersByTime(6 * TWO_SIDED_PAGE_PRINT_TIME_MS + 1);
+    } else {
+      jest.advanceTimersByTime(LAST_PRINT_JOB_SLEEP_MS + 1);
+    }
   }
+
+  await screen.findByText('All Precincts');
+  expect(screen.queryByText('Printing')).not.toBeInTheDocument();
+});
+
+test('Printing L&A package for one precinct, when HMPBs are not letter-size', async () => {
+  const electionWithLegalSizeHmpbsDefinition = _.cloneDeep(
+    electionWithMsEitherNeitherDefinition
+  );
+  electionWithLegalSizeHmpbsDefinition.election.ballotLayout!.paperSize =
+    BallotPaperSize.Legal;
```

Comment:
I'd rather to this:
```ts
  const electionWithLegalSizeHmpbsDefinition = asElectionDefinition({
    ...electionWithMsEitherNeitherDefinition.election,
    ballotLayout: {
      paperSize: BallotPaperSize.Legal,
    },
  });
```
Which doesn't use `_.cloneDeep` or a non-null assertion and has `electionData` and `electionHash` that actually match the `election`.